time-calculator/time_calculator.py

```diff
@@ -0,0 +1,67 @@
+
+def add_time(startTime, stopTime, dayOfWeek=None):
+
+    # Start Time
+    [startTime, start_am_pm] = startTime.split()
+    [startHours, startMinutes] = startTime.split(':')
+
+    # End Time
+    [stopHours, stopMinutes] = stopTime.split(":")
+
+    stopMinutes = int(stopMinutes) + (60 * int(stopHours))
+
+    if start_am_pm == "PM":
+        startHours = int(startHours) + 12
+
+    startMinutes = int(startMinutes) + (60 * int(startHours))
+    totalMinutes = startMinutes + stopMinutes
+
+    # Minutes Calculation
+    finalMinutes = totalMinutes % 60
+    finalHours = int(totalMinutes / 60)
+
+    if len(str(finalMinutes)) == 1:
```

Comment:
Instead of `len(str())` we can do it in simpler way i.e. just compare the number directly